sqlalchemy_bigquery/base.py

```diff
@@ -1057,7 +1068,10 @@ def __init__(self, *args, **kwargs):
             raise TypeError("The unnest function requires a single argument.")
         arg = args[0]
         if isinstance(arg, sqlalchemy.sql.expression.ColumnElement):
-            if not isinstance(arg.type, sqlalchemy.sql.sqltypes.ARRAY):
+            if not (
+                isinstance(arg.type, sqlalchemy.sql.sqltypes.ARRAY)
+                or isinstance(arg.type.impl, sqlalchemy.sql.sqltypes.ARRAY)
```

Comment:
hey @sharoonthomas, 
```python
or isinstance(arg.type.impl, sqlalchemy.sql.sqltypes.ARRAY)
```
is causing a unit test to fail with `AttributeError: 'String' object has no attribute 'impl'`